### PR TITLE
Add new component renault documentation

### PR DESCRIPTION
**Pull request by epenet**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/39605
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1825
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by epenet** [comment]
@frenck you asked on the main PR that I remove the extra platforms (binary_sensor, climate, device tracker) for the new integration, and follow up with subsequent PRs.
For the documentation is it best to keep in sync with the main PR, or set the documentation PR as we hope it will be when all the subsequent PRs are merged?

**Comment by frenck** [comment]
The documentation must match the linked PR

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by epenet** [comment]
Hi @frenck,

Following the discussions on the main PR, I have amended the documentation with a link to the instructions on how to grab the API keys from the Renault servers.

**Review by frenck — DISMISSED** [review]

File: source/_integrations/renault.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Renault
+description: Instructions on how to integrate Renault car into Home Assistant.
+ha_category:
+  - Car
+  - Sensor
+ha_release: 0.115
```
Comment (milanmeu) [inline source/_integrations/renault.markdown]:
```suggestion
ha_release: 2021.8
```

**Review by milanmeu — COMMENTED** [review]
@frenck Parent merged.
This should be merged to the current branch because the integration is already available in 2021.8.

**Review by frenck — APPROVED** [review]

**Pull request merged**